ftw/simplelayout/browser/client/dist/simplelayout.js

```diff
@@ -635,6 +635,13 @@ define('app/simplelayout/Block',[
       return this;
     };
 
+    this.markLinks = function() {
```

Comment:
All other plone integration function are in the main.js?
We wait for @bierik ...